platform/email/platform-email-impl/src/main/java/org/codice/ddf/platform/email/impl/SmtpClientImpl.java

```diff
@@ -125,19 +128,22 @@ public Session createSession() {
   }
 
   @Override
-  public Future<Void> send(Message message) throws MessagingException {
+  public Future<Void> send(Message message) {
     notNull(message, "message must be non-null");
     return executorService.submit(
-        new Callable<Void>() {
-          @Override
-          public Void call() throws Exception {
+        () -> {
+          try {
             Transport.send(message);
-            SecurityLogger.audit(
-                "Sent an email: recipient={} subject={}",
-                Arrays.toString(message.getAllRecipients()),
-                message.getSubject());
+          } catch (MessagingException e) {
+            LOGGER.debug("Could not send message {}", message, e);
             return null;
```

Comment:
Since we're removing the exception should we return a boolean from this future (instead of void) to indicate success or failure.

Of course we could creep scope ever further and then update the usages (in FeedbackApplication and EmailNotifier) no react to whether they pass or fail and change the status code accordingly. Though currently it looks like its a fire-and-forget so that may be a sizable change

Reply:
The exception was never percolated up, so the indication of success or failure was lost anyway (which necessitated this change).